.github/workflows/ci-circleci.yml

```diff
@@ -0,0 +1,19 @@
+on:
+  push:
+    branches: [master, "release/*"]
+  pull_request:
+    branches: [master, "release/*"]
+    paths:
+      - "requirements/pytorch/**"
+      - "src/pytorch_lightning/**"
+      - "tests/tests_pytorch/**"
+      - "setup.cfg"  # includes pytest config
+
+jobs:
+  # https://circleci.com/blog/trigger-circleci-pipeline-github-action
+  trigger-circleci:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: circleci/trigger_circleci_pipeline@v1.0
+        env:
+          CCI_TOKEN: ${{ secrets.CCI_TOKEN }}
```

Comment:
@Borda I need this token. Do you know how to get it?

info: https://circleci.com/blog/trigger-circleci-pipeline-github-action